### PR TITLE
Add Changelog for 3.0.3

### PR DESCRIPTION
**Pull request by Stexxe**

### CODE REVIEW

**Review by bjhham — APPROVED** [review]

**Comment by osipxd** [comment]
Should we also merge #4559 and #4556?

**Comment by Stexxe** [comment]
> Should we also include these PRs?
> 
> * [KTOR-7952 Fix for uninitialized writerJob property #4559](https://github.com/ktorio/ktor/pull/4559)
> * [KTOR-7963 Handle ClosedWatchServiceException #4556](https://github.com/ktorio/ktor/pull/4556)

@bjhham

**Review by bjhham — APPROVED** [review]

**Comment by Stexxe** [comment]
> > Should we also include these PRs?
> > 
> > * [KTOR-7952 Fix for uninitialized writerJob property #4559](https://github.com/ktorio/ktor/pull/4559)
> > * [KTOR-7963 Handle ClosedWatchServiceException #4556](https://github.com/ktorio/ktor/pull/4556)
> 
> @bjhham

These fixes will go to the next release

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+### Bugfixes
+* "Module not found" errors when executing browserProductionWebpack task since 3.0.2 ([KTOR-7912](https://youtrack.jetbrains.com/issue/KTOR-7912))
+* Darwin: "IllegalStateException: Content-Length mismatch" on requesting gzipped content ([KTOR-7943](https://youtrack.jetbrains.com/issue/KTOR-7943))
+* JS/WASM fails with "IllegalStateException: Content-Length mismatch" on requesting gzipped content ([KTOR-7934](https://youtrack.jetbrains.com/issue/KTOR-7934))
+* FormFieldLimit is overwritten by default arg ([KTOR-7946](https://youtrack.jetbrains.com/issue/KTOR-7946))
+* A Performance issue reading with ByteReadChannel.readUTF8LineTo request body ([KTOR-7941](https://youtrack.jetbrains.com/issue/KTOR-7941))
+* Installing HttpCache before ContentEncoding prevents response body to be decoded ([KTOR-7830](https://youtrack.jetbrains.com/issue/KTOR-7830))
+* TestApplication.stop() doesn't stop the application anymore ([KTOR-7682](https://youtrack.jetbrains.com/issue/KTOR-7682))
+* File is not commited after closing writeChannel() of the file ([KTOR-7845](https://youtrack.jetbrains.com/issue/KTOR-7845))
+
```
Comment (bjhham) [inline CHANGELOG.md:13]:
```suggest
* [KTOR-7952](https://youtrack.jetbrains.com/issue/KTOR-7952) SaveBodyPlugin: UninitializedPropertyAccessException when reading response body within receivePipeline
* [KTOR-7963](https://youtrack.jetbrains.com/issue/KTOR-7963) Uncaught ClosedWatchServiceException exception thrown by finalizer when closing the server

```

Comment (Stexxe) [reply CHANGELOG.md:13]: These fixes will be included in the next release because the artifacts had already been closed in Maven.

**Pull request merged**